ru.sview.sView.appdata.xml

```diff
@@ -30,6 +30,6 @@
   </screenshots>
   <update_contact>hadess@hadess.net</update_contact>
   <releases>
-    <release version="20.08" date="2020-08-02"/>
+    <release version="23.05" date="2023-03-01"/>
```

Comment:
There's no 23.05 release, it's not a tag, or a release:
https://github.com/gkv311/sview/releases